doc/stepbystep/getting_started_with_openCV.md

```diff
@@ -9,36 +9,34 @@ Using the editor of your choice create BGR_sample.cpp and copy-paste the followi
 ```cpp
 // include the librealsense C++ header file
 #include <librealsense2/rs.hpp>
-#include <librealsense2/rsutils.hpp>
 
 // include OpenCV header file
 #include <opencv2/opencv.hpp>
 
 using namespace std;
 using namespace cv;
-using namespace rs2;
 
 int main()
 {
-    // Create a context object. This object owns the handles to all connected realsense devices
-    context ctx;
+    //Contruct a pipeline which abstracts the device
+    rs2::pipeline pipe;
+    //Create a configuration for configuring the pipeline with a non default profile    
+    rs2::config cfg;
 
-    // Access the first available RealSense device
-    device dev = ctx.query_devices().front();
+    //Add desired streams to configuration
+    cfg.enable_stream(RS2_STREAM_INFRARED, 640, 480, RS2_FORMAT_Y8, 30);
+    cfg.enable_stream(RS2_STREAM_DEPTH, 640, 480, RS2_FORMAT_Z16, 30);
 
-    util::config config;
-    config.enable_stream(RS2_STREAM_COLOR, 640, 480, 30, RS2_FORMAT_BGR8);
-    auto stream = config.open(dev);
+    //Instruct pipeline to start streaming with the requested configuration:
+    pipe.start(cfg);
 
-    frame_queue queue;
-    // Start streaming
-    stream.start(queue);
+    //Wait for all configured streams to produce a frame
+    rs2::frameset frames = pipe.wait_for_frames();
 
-    // Camera warmup - Dropped several first frames to let auto-exposure stabilize
-    for(int i = 0; i < 30; i++)
-       queue.wait_for_frame();
+    //Get each frame 
+    rs2::frame ir_frame = frames.first(RS2_STREAM_INFRARED);
+    rs2::frame depth_frame = frames.get_depth_frame();
 
-    frame f = queue.wait_for_frame();
     // Creating OpenCV Matrix from a color image
     Mat color(Size(640, 480), CV_8UC3, (void*)f.get_data(), Mat::AUTO_STEP);
```

Comment:
This won't compile with (void*)f....
You need to add render of the second stream IR (if you capture two streams) and add the description in line 41